sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/BadCredentialsException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.authn.jwt;
+
+public class BadCredentialsException extends Exception {
+
+    private static final long serialVersionUID = 9092575587366580869L;
```

Comment:
Is this representing core? It seems like it is a dead variable

Reply:
See discussion on SO about `serialVersionUID`: https://stackoverflow.com/a/285809

This initial implementation was inspired by JWTs within the security plugin and this class is a copy and admittedly this should changed since the fully-qualified class name has changed.

Doing a quick search of the security repo: https://github.com/opensearch-project/security/search?p=1&q=serialVersionUID

It looks like exceptions related to auth have this value set, but I'm not exactly sure why.